src/api/worker/offline/OfflineStorage.ts

```diff
@@ -116,11 +117,24 @@ export class OfflineStorage implements CacheStorage, ExposedCacheStorage {
 		// We open database here, and it is closed in the native side when the window is closed or the page is reloaded
 		await this.sqlCipherFacade.openDb(userId, databaseKey)
 		await this.createTables()
-		await this.migrator.migrate(this, this.sqlCipherFacade)
+		try {
+			await this.migrator.migrate(this, this.sqlCipherFacade)
+		} catch (e) {
+			if (!(e instanceof OutOfSyncError)) {
```

Comment:
can we have `if`/`else` instead please, this is very confusing to read right now